content/blog/newsletter/2024-04-05-newsletter.md

```diff
@@ -48,7 +48,7 @@ The XSF offers [fiscal hosting](https://xmpp.org/community/fiscalhost/) for XMPP
 
 ## XMPP Events
 
-- [16th May 2024: XMPP Italian happy hour](https://tube.nicfab.eu/c/xmpp) [IT]: monthly Italian XMPP web meeting, starting May 16th and then every third Monday of the month at 7:00 PM local time (online event, with web meeting mode and live streaming).
+- [XMPP Italian happy hour](https://tube.nicfab.eu/c/xmpp) [IT]: monthly Italian XMPP web meeting, every third Monday of the month at 7:00 PM local time (online event, with web meeting mode and live streaming).
```

Comment:
sorry, wrong URL. The correct url is: https://video.xmpp-it.net/c/happyhour/videos